/aruco_node:
  ros__parameters:
    marker_size: 1.0
#    marker_size: 0.055
#    aruco_dictionary_id: DICT_5X5_250
    aruco_dictionary_id: DICT_6X6_250
#    image_topic: /image_raw
    image_topic: /drone/camera/image_raw
#    camera_info_topic: /camera_info
    camera_info_topic: /drone/camera/camera_info

